gradle.properties

```diff
@@ -1,5 +1,5 @@
 pluginVersion=2.0.3
-pluginSinceBuild=202
+pluginSinceBuild=201
```

Comment:
We have two plugin versions deployed in JB plugin portal: first is until 2020.1 (201) included and the second is from 2020.2 (202+). So we cannot just decrease compatibility matrix.
I guess this change is because of Android Studio (based on 2020.1). What I suggest whether wait for a new AS based on 2020.2 (should be available earlier next year), or build s specific version with this changes locally.

Reply:
Thanks @pavel-snyk 

Got it! I thought it would be great to backport this feature for current Snyk users in Android Studio. I understand the compatibility matrix issue and will revert the change in an update

Reply:
👍.
p.s: I checked release cycle by AS, it looks like next version would be Jan-Feb 2021.